scripts/linters/test_files/invalid_copyright.py

```diff
@@ -22,7 +22,7 @@
 from core import python_utils
 
 
-class FakeClass(python_utils.OBJECT):
```

Comment:
Remove `python_utils` import from this and other files below if it is not used anywhere else.